src/utils/getMaxNumberId.tsx

```diff
@@ -0,0 +1,7 @@
+import { Todo } from '../types';
+
+export const getMaxNumberId = (todos: Todo[]): number => {
+  const result: number[] = todos.map(todo => todo.id);
+
+  return Math.max(...result) + 1;
```

Comment:
This code assumes that there's at least one todo in the array. If the array is empty, `Math.max(...result)` will return `-Infinity`, and the function will return `0`. Consider handling the case when `todos` is empty to avoid potential bugs.

Reply:
yep, u have a bug in this function

Reply:
fix it